src/components/PostEditor/PostEditor.js

```diff
@@ -425,16 +433,28 @@ export default class PostEditor extends React.Component {
             />
           </div>
         </div>}
-        {isProject && currentUser.hasFeature(PROJECT_CONTRIBUTIONS) && <div styleName='footerSection'>
-          <div styleName='footerSection-label'>Accept Contributions</div>
-          {hasStripeAccount && <div styleName={cx('footerSection-communities', 'accept-contributions')}>
-            <Switch value={acceptContributions} onClick={this.toggleContributions} styleName='accept-contributions-switch' />
-            {!acceptContributions && <div styleName='accept-contributions-help'>If you turn "Accept Contributions" on, people will be able to send money to your Stripe connected account to support this project.</div>}
-          </div>}
-          {!hasStripeAccount && <div styleName={cx('footerSection-communities', 'accept-contributions-help')}>
-            To accept financial contributions for this project, you have to connect a Stripe account. Go to <a href='/settings/payment'>Settings</a> to set it up. (Remember to save your changes before leaving this form)
-          </div>}
-        </div>}
+        <div styleName='footerSection'>
+          <div styleName='footerSection-label'>Topics</div>
+          <div styleName='footerSection-communities'>
+            <TopicSelector
+              selectedTopics={topics}
+              detailsTopics={detailsTopics}
+              ref={this.topicSelector} />
+          </div>
+        </div>
+        <div styleName='footerSection'>
+          <div styleName='footerSection-label'>Post in</div>
+          <div styleName='footerSection-communities'>
+            <CommunitiesSelector
+              options={communityOptions}
+              selected={communities}
+              onChange={this.setSelectedCommunities}
+              readOnly={loading}
+              ref={this.communitiesSelector}
+            />
+          </div>
+        </div>
+        <PublicToggle togglePublic={this.togglePublic} isPublic={post.isPublic ? post.isPublic : false} />
```

Comment:
one sexy way to do `post.isPublic ? post.isPublic : false` is !!isPublic

Reply:
💯